src/js/structs/UniversePackage.js

```diff
@@ -22,6 +22,10 @@ class UniversePackage extends Item {
     );
   }
 
+  hasKnownIssues() {
+    return this.get("hasKnownIssues");
```

Comment:
We don't have this struct in `PackageDetailTab` yet.

Reply:
really? how does the integration test pass then?

Reply:
As far as I can tell, currently we always show the warning.

Reply:
i think we don't. at least that was the intention behind this test: https://github.com/dcos/dcos-ui/pull/4092/files#diff-887b4fcfde47a3104c12bf49efea886aR125

Reply:
Where do we check for known issues?

Reply:
wow! 🤦‍♂ 	thanks!

Reply:
```diff
diff --git a/src/js/pages/catalog/PackageDetailTab.js b/src/js/pages/catalog/PackageDetailTab.js
index d6b99feff..84399e71a 100644
--- a/src/js/pages/catalog/PackageDetailTab.js
+++ b/src/js/pages/catalog/PackageDetailTab.js
@@ -626,6 +626,7 @@ class PackageDetailTab extends mixin(StoreMixin) {
                   {!state.isLoadingSelectedVersion && (
                     <div className="media-object-item media-object-item-grow ">
                       <h1 className="short flush-top flush-bottom">
+                        {cosmosPackage.hasKnownIssues() ? (
                           <Box display="inline-block">
                             <UIKitTooltip
                               id="has-known-issues"
@@ -637,11 +638,12 @@ class PackageDetailTab extends mixin(StoreMixin) {
                               }
                             >
                               <Trans render="span">
-                              This package is known to have issues when started
-                              with its default settings.
+                                This package is known to have issues when
+                                started with its default settings.
                               </Trans>
                             </UIKitTooltip>
-                        </Box>{" "}
+                          </Box>
+                        ) : null}
                         {name}
                       </h1>
                       <div className="flex flex-align-items-center">
diff --git a/tests/pages/catalog/packages/package/PackageTab-cy.js b/tests/pages/catalog/packages/package/PackageTab-cy.js
index 404c34a77..de6715229 100644
--- a/tests/pages/catalog/packages/package/PackageTab-cy.js
+++ b/tests/pages/catalog/packages/package/PackageTab-cy.js
@@ -122,6 +122,7 @@ describe("Package Detail Tab", function() {
     });
 
     it("Does not shows a knownIssues-warning ", () => {
+      cy.get("h1").contains("marathon");
       cy.get("[aria-describedby=has-known-issues]").should("not.exist");
     });
 

```